### DISCUSSION TITLE
What kind of creature was Legolas in The Lord of the Rings?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#715] [Galaxy Trivia] [answered]
**Bonus:** How old is he? (*answer in lucky 22 base notation*)

**Comment by SANKARAMDAS** [comment]
Legolas is a Sindarin Elf from the Woodland Realm of Northern Mirkwood.
The Lord of the Rings list Legolas as being born in the year 87 of the Third Age of Middle-earth, which makes him some 2,931 years old around the time of the War of the Ring, which happened over 3,000 years into the Third Age of Middle-earth.

**Comment by thuongtruong1009** [comment]
Legolas, a prominent character in J.R.R. Tolkien's "The Lord of the Rings," is an elf from the woodland realm of Mirkwood.

Regarding Legolas' age, his exact age is not explicitly mentioned in the books. However, it is known that he is a Sindarin Elf, and Sindarin Elves are known for having long lifespans. In the book, Legolas is described as being a "young" elf, but Tolkien does not provide specific details on his age.

Since you requested the answer in "lucky 22 base notation," which implies using a base-22 numeral system, I cannot convert his age directly because I don't have his age in decimal notation. However, if you provide the age in decimal notation, I can attempt to convert it to base-22 for you.

**Comment by peterbaccaro** [comment, marked as answer]
Legolas is a **Sindarin Elf** from the Woodland Realm of Northern Mirkwood.

Legolas was portrayed by Orlando Bloom. In the "official movie guide" for The Lord of the Rings, a birthdate for Legolas is set to 87 of the Third Age. This would make him 2931 years old at the time of the War of the Ring. In base 22 notation this would be **615**.
